x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -20,17 +20,17 @@ public class BlobCacheMetrics {
     public BlobCacheMetrics(MeterRegistry meterRegistry) {
         this(
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.miss_that_triggered_read",
+                "es.blob_cache.miss_that_triggered_read.count",
                 "The number of times there was a cache miss that triggered a read from the blob store",
                 "count"
             ),
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.count_of_evicted_used_regions",
+                "es.blob_cache.evicted_used_regions.count",
```

Comment:
Same (possibly `total`?)

Reply:
count and total feel very similar to me. I would use total for async counter or gauge (reporting a cumulative value)
and count for deltas like here.

Reply:
changed to total